test/active_record_cache_find_by_test.rb

```diff
@@ -150,8 +155,9 @@ def assert_not_caches(key, cache = Rails.cache)
   end
 
   it('caches find_by_x with two attributes') do
+    skip 'was not cached on rails 4.2'
```

Comment:
@notEthan did you say we could remove all of this?

doesn't AR already have caching ?